rlmeta/core/remote.py

```diff
@@ -26,15 +26,25 @@ def __new__(cls, name, bases, attrs):
             if getattr(method, "__remote__", False):
                 remote_methods.add(method.__name__)
         attrs["__remote_methods__"] = list(remote_methods)
+        attrs["__identifier__"] = ""
         return super().__new__(cls, name, bases, attrs)
 
 
 class Remotable(abc.ABC, metaclass=RemotableMeta):
+    def __init__(self, identifier=""):
```

Comment:
A high level comment is we have to also modify the __init__ method for the classes derived from Remotable. Or at least add a TODO here.